### ISSUE TITLE
about the definition of semantic entropy

### ISSUE THREAD

**Issue by wangyifei0047** [shuoli90/Rank-Calibration#1]
"Sorry to disturb,  I am a little confused about  the code for computing semantic entropy did not include a probability mass?"
source code: 
entropy = - torch.sum(aggregated_log_likelihoods, dim=0) / torch.tensor(aggregated_log_likelihoods.shape[0])
Thanks!

**Comment by shuoli90** [comment]
Hi Wangyifei0047, 

Thanks for your question. I am happy to discuss. 

I am not sure if I fully understand your question. But let me kick start the discussion. We implemented the semantic entropy according to Equation.4 in this work: [semantic entropy](https://arxiv.org/pdf/2302.09664). 

That being said, we did include the probability mass, named as _aggregated_log_likelihoods_. These are probability masses for individual semantic clusters. 

Does this make sense to you? Feel free to post further discussions or questions.
